DotNETDepends/SolutionReader.cs

```diff
@@ -252,6 +253,7 @@ private async Task ProcessProjectAsync(ProjectId projectId, Solution solution, P
             var solutionRoot = Path.GetDirectoryName(solution.FilePath);
             if (project != null && project.FilePath != null && solutionRoot != null)
             {
+                Console.WriteLine("Processing project: " + project.FilePath);
```

Comment:
**question**: For the dependency tools, don't we expect them to write out the results to stdout? Should we be using some kind of "stderr" type stream for writing debugging?

Reply:
Unfortunately some of the other tools and APIs we call already write to stdout.  I solved this by adding a marker in the output when we actually start to write the resulting JSON.  This won't hurt that.